commit 8c8d11dbccc63d78788d708dace8248ee6b80733 (refs/changes/12/2912/3)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2017-11-06T17:06:38+00:00 (1 year, 11 months ago)
    
    OS-6442 new LTP failures
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/usr/src/lib/brand/lx/testing/ltp_skiplist b/usr/src/lib/brand/lx/testing/ltp_skiplist
index d4195353cc..91e9cc5cca 100644
--- a/usr/src/lib/brand/lx/testing/ltp_skiplist
+++ b/usr/src/lib/brand/lx/testing/ltp_skiplist
@@ -132,6 +132,7 @@ clone09
 crash02
 dirtyc0w
 fallocate04		# needs SEEK_HOLE
+fallocate05		# needs to mount a filesystem
 fanotify01
 fanotify02
 fanotify03
@@ -157,6 +158,7 @@ fork05
 fork09
 fork13
 fork14
+fsync01
 ftruncate04
 ftruncate04_64
 futex_wake04		# OS-4471
@@ -238,6 +240,7 @@ setpgid02
 setregid02
 setregid02_16
 setrlimit01
+setsockopt02
 setxattr01
 setxattr02
 setxattr03
